graftm/create.py

```diff
@@ -747,8 +747,7 @@ def main(self, **kwargs):
                 tre_file_tempfile = tempfile.NamedTemporaryFile(suffix='.tree', prefix='graftm')
                 tre_file = tre_file_tempfile.name
                 with tempfile.NamedTemporaryFile(suffix='.tree', prefix='graftm') as f:
-                    tree.write(f)
-                    f.flush()
+                    tree.write(f.name)
```

Comment:
we are giving `TreeNode.write` a string path, not an IO object. So it does open, write, and close for us. The context (if that is the word) opens f as an IO object, but actually we don't need that, we just want the tempfile. But writing it like this is convenient so that the tempfile is deleted at the end of the context.